kit/src/main/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMDocument.java

```diff
@@ -102,13 +125,7 @@ public FXOMDocument(String fxmlText, URL location, ClassLoader classLoader, Reso
 
         hasGluonControls = fxmlText.contains(EditorPlatform.GLUON_PACKAGE);
     }
-    
-    
-    public FXOMDocument(String fxmlText, URL location, ClassLoader classLoader, ResourceBundle resources) throws IOException {
-        this(fxmlText, location, classLoader, resources, true /* normalize */);
```

Comment:
Hmm, I realize now that you have changed the default behaviour of all classed that called `FXOMDocument` via this removed constructor, where `normalize` was set to true: since they don't use switches now (except PreviewWindowController), therefore `normalized` is set to false but it used to be true. Only exception that works is FXOMRefresher (as it used the other constructor).

The only possible fix I see now is to add NORMALIZED to: EditorControler, AddContextMenuToSelectionJob, AddTootiplToSelectionJob, ImportWindowController,...

Reply:
Yes, the original intent of having the `NON_NORMALIZED` item was token the constructors default behavior. There are no tests at many places and my knowledge how things are supposed to work is very limited, hence I tried to make this change as minimal in first. Reworking it to `NORMALIZED` now feels as an avoidable mistake.

Reply:
Following locations require an update:

Kit:
* `com.oracle.javafx.scenebuilder.kit.fxom.FXOMSaverUpdateImportInstructionsTest` line 286
* `com.oracle.javafx.scenebuilder.kit.preview.PreviewWindowController` lines 215, 282, were already okay
* `com.oracle.javafx.scenebuilder.kit.metadata.util.ClipboardDecoder` line 92, required `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.library.LibraryItem` line 90 , required `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.fxom.FXOMRefresher` line 63 , no switches
* `com.oracle.javafx.scenebuilder.kit.fxom.FXOMNodes` lines 155 and 207, required `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.fxom.FXOMArchive` line 76, required `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.editor.EditorController` line 2561, requires `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.editor.panel.library.LibraryPanelController` line 1004, requires `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.editor.panel.ImportWindowController` line561, requires `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.editor.job.AddTooltipToSelectionJob`line 136, requires `NORMALIZED`
* `com.oracle.javafx.scenebuilder.kit.editor.job.AddContextMenuToSelectionJob`line 136, requires `NORMALIZED`

This is all places I have identified so far.

Reply:
I've provided a commit with the necessary changes.

Reply:
Okay, thanks.